ktor-client/ktor-client-okhttp/jvm/test/io/ktor/client/engine/okhttp/RequestTests.kt

```diff
@@ -32,4 +52,32 @@ class RequestTests {
             client.get<String>("https://google.com")
         }
     }
+
+    @Test
+    fun testReusingRequestBuilderOnMultipleClients() {
+        val rb = HttpRequestBuilder()
+        rb.url.takeFrom("http://0.0.0.0:$serverPort/delay?delay=500")
```

Comment:
Consider using `$TEST_URL` constant

Reply:
can't be constant because it uses `$serverPort`, but extracted to read-only property in the base class